trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.test.ts

```diff
@@ -212,11 +213,73 @@ describe('controllers/utilisation-reports/add-to-an-existing-payment', () => {
       expect((res._getRenderData() as AddToAnExistingPaymentViewModel).reportId).toEqual('123');
     });
 
+    it('should set the back link href', async () => {
+      // Arrange
+      const { req, res } = httpMocks.createMocks({
+        session: requestSession,
+        params: { reportId },
+        body: {
+          'feeRecordIds-1,22-reportedPaymentsCurrency-USD-status-TO_DO': 'on',
+          'feeRecordIds-333-reportedPaymentsCurrency-USD-status-TO_DO': 'on',
+        },
+      });
+      jest.mocked(api.getSelectedFeeRecordsDetailsWithAvailablePaymentGroups).mockResolvedValue({
+        ...aSelectedFeeRecordsDetails(),
+        feeRecords: [
+          {
+            ...aSelectedFeeRecordDetails(),
+            id: 1,
+            reportedFee: { amount: 2000, currency: 'EUR' },
+          },
+          {
+            ...aSelectedFeeRecordDetails(),
+            id: 22,
+            reportedFee: { amount: 3000, currency: 'EUR' },
+          },
+          {
+            ...aSelectedFeeRecordDetails(),
+            id: 333,
+            reportedFee: { amount: 100, currency: 'JPY' },
+          },
+        ],
+      });
+
+      // Act
+      await addToAnExistingPayment(req, res);
+
+      // Assert
+      expect((res._getRenderData() as AddToAnExistingPaymentViewModel).backLinkHref).toEqual(
```

Comment:
I think this should just be `.toEqual(getLinkToPremiumPaymentsTab(reportId))` rather than asserting the result of `getLinkToPremiumPaymentsTab`